config/routes.rb

```diff
@@ -2,9 +2,8 @@
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
-  # root :to => 'users#index'
+  # root "articles#index"
   resources :users, only: [:index, :show] do 
     resources :posts, only: [:index, :show]
   end
```

Comment:
- [ ] You should add comments and likes as routes here. Each comment and like belongs to a post. In this way, you will clear the unnecessary method in your post controller and you only create an instance variable by accessing the parameters.